test/server/cards/01_SOR/events/ForceThrow.spec.ts

```diff
@@ -0,0 +1,76 @@
+describe('Force Throw', function() {
+    integration(function(contextRef) {
+        describe('Force Throw\'s ability', function() {
+            it('should select opponent to discard a card and deal damage to a unit equals to its cost', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['force-throw'],
+                        groundArena: ['wampa', 'ezra-bridger#resourceful-troublemaker']
+                    },
+                    player2: {
+                        hand: ['karabast'],
+                        groundArena: ['specforce-soldier', 'atst'],
+                        spaceArena: ['tieln-fighter']
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // Damage dealt by opponent discarded card
+                context.player1.clickCard(context.forceThrow);
+                context.player1.clickPrompt('Opponent');
```

Comment:
Can we add a check that the right set of buttons appears using `.toHaveExactPromptButtons()`